pod/perldiag.pod

```diff
@@ -2275,6 +2275,12 @@ as a goto, or a loop control statement.
 (W exiting) You are exiting a substitution by unconventional means, such
 as a return, a goto, or a loop control statement.
 
+=item Expected a %s reference in lexically_export
+
+(F) A call to L<builtin/lexically_export> associates new names with items
+given by reference.  Every second argument must be a reference to a kind
+of item that matches the sigil in the name it is to be associated with.
```

Comment:
The general style is to first explain what you did wrong, then explain why. Maybe something like:

> The type of a reference given to L<builtin/lexically_export> did not match the sigil of the preceding name.